static/app/views/issueDetails/traceTimeline/traceTimeline.spec.tsx

```diff
@@ -259,29 +268,28 @@ describe('TraceTimeline', () => {
     expect(useRouteAnalyticsParams).toHaveBeenCalledWith({
       has_related_trace_issue: false,
       trace_timeline_status: 'empty',
-      trace_timeline_two_issues: false,
     });
   });
 
   it('works for plans with no global-views feature', async () => {
+    // This test will call the endpoint without the global-views feature, thus,
+    // we will only look at the current project (project: event.projectID) instead of passing -1
```

Comment:
The two comments below were a bit unclear to me. This is a bit more elaborate and hopefully clearer to future readers.